data/json/items/vehicle/rotor.json

```diff
@@ -4,26 +4,42 @@
     "type": "GENERIC",
     "category": "veh_parts",
     "name": { "str": "heavy-duty military rotors", "str_pl": "sets of heavy-duty military rotors" },
-    "description": "A set of four rotor blades from a military attack helicopter.",
+    "description": "A large four blade rotor assembly from a military attack helicopter.",
     "price": 23400,
     "price_postapoc": 500,
-    "weight": "452000 g",
-    "volume": "452000 ml",
+    "weight": "1125000 g",
+    "//": "Increased weight from ~1000lbs to ~2500lbs to account for the mast, swashplates, transmission, pumps, and all other relevant parts of rotor assembly besides the blades",
+    "volume": "1125000 ml",
```

Comment:
Although there's no chance to obtain this item "as an item", but being larger than 1000L will prevent it from being spawned via debug. However, I don't think there's any other impact.

Reply:
I edited the weight and volume down to 1000kg and 1000L to fix the spawning issue. The change doesn't make much difference for the Blackhawk (250lbs difference to a 10,000lbs vehicle), and these weight/volume figures are educated guesses anyway.